tests/pr_test.go

```diff
@@ -6,14 +6,17 @@ import (
 	"os"
 	"testing"
 
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testschematic"
```

Comment:
@marifse maybe try moving this as the last import? Just a guess, but that works in [this test](https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/blob/bcdb164522ff654f674fa7bef717a97f5b1aa5f9/tests/pr_test.go#L4-L13)